src/test/kotlin/io/github/mojira/arisa/infrastructure/IntegrationTest.kt

```diff
@@ -13,7 +13,8 @@ class IntegrationTest : StringSpec({
                 mapOf(
                     "arisa.credentials.username" to "test",
                     "arisa.credentials.password" to "test",
-                    "arisa.credentials.dandelionToken" to "test"
+                    "arisa.credentials.dandelionToken" to "test",
+                    "arisa.credentials.discordLogWebhook" to "test"
```

Comment:
I would make it optional and test what happens when we dont include it(shouldnt log to discoerd if its not there)

Reply:
`discordLogWebhook` is required